templates/default/disable_ipv6_loopback.sh.erb

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Disable the ipv6 loopback address to make chef server and EMPD happy
+if ! grep net.ipv6.conf.lo.disable_ipv6 /etc/sysctl.conf; then
+  echo "net.ipv6.conf.lo.disable_ipv6=0" | sudo tee -a /etc/sysctl.conf
```

Comment:
Note, enabling ipv6 is indeed what we want if the system supports IPV6 addr types.